src/VisualStudio/Core/Next/Remote/RemoteHostClientFactory.cs

```diff
@@ -9,6 +9,8 @@
 
 namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
+    using Workspace = Microsoft.CodeAnalysis.Workspace;
```

Comment:
Yes. I will try to convert the other reference to an extern alias to avoid this.